### PR TITLE
Tests and actions

### PR DESCRIPTION
**Pull request by laves**

### CODE REVIEW

File: test/VoiceProcessorTestApp/.editorconfig [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# EditorConfig helps developers define and maintain consistent
+# coding styles between different editors and IDEs
+# editorconfig.org
+
+root = true
+
+[*]
+
+indent_style = space
+indent_size = 4
```
Comment (ksyeo1010) [inline test/VoiceProcessorTestApp/.editorconfig]: don't we do 2 spaces for js/ts?

Comment (laves) [reply test/VoiceProcessorTestApp/.editorconfig]: yeah, but the problem with using that with react native is it doesn't mix with the java codestyle conventions which require 4 to pass. I'll try to find a way to support both.

Comment (ksyeo1010) [reply test/VoiceProcessorTestApp/.editorconfig]: ok, if its too much its fine as is.

Comment (laves) [reply test/VoiceProcessorTestApp/.editorconfig]: I removed editorconfigs and just let prettier, xcode and android studio do their own formatting. Works better this way.

**Review by ksyeo1010 — APPROVED** [review]

**Pull request merged**